File: primefinder/Prime.java
public class Prime {

    static boolean isPrime(int number) {
        if (number <= 1) {
            return false;
        }

        for (int i = 2; i * i <= number; i++) {
            if (number % i == 0) {
                return false;
            }
        }

        return true;
    }

    public static void main(String[] args) {
        int start = 1, end = 10000000;
        int primeCount = 0;

        long startTime = System.currentTimeMillis();

        System.out.println("Prime numbers in the range " + start + " to " + end + " are:");
        for (int i = start; i <= end; i++) {
            if (isPrime(i)) {
                primeCount++;
            }
        }

        long endTime = System.currentTimeMillis();

        double timeTaken = (endTime - startTime) / 1000.0;
        System.out.println("Time taken: " + timeTaken + " seconds");
        System.out.println("Total number of prime numbers in the range: " + primeCount);
    }
}

File: primefinder/prime.js
function isPrime(number) {
  if (number <= 1) {
    return false;
  }

  for (let i = 2; i * i <= number; i++) {
    if (number % i === 0) {
      return false;
    }
  }

  return true;
}

const start = 1;
const end = 100000000;
let primeCount = 0;

const startTime = new Date().getTime() / 1000;

console.log(`Prime numbers in the range ${start} to ${end} are:`);
for (let i = start; i <= end; i++) {
  if (isPrime(i)) {
    primeCount++;
  }
}

const endTime = new Date().getTime() / 1000;

const timeTaken = endTime - startTime;
console.log(`Time taken: ${timeTaken} seconds`);
console.log(`Total number of prime numbers in the range: ${primeCount}`);
